benchmarks/cugraph/standalone/bulk_sampling/bulk_sampling.sh

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export RAPIDS_NO_INITIALIZE="1"
+export CUDF_SPILL="1"
+export LIBCUDF_CUFILE_POLICY=OFF
+
+dataset_name="ogbn_papers100M"
+dataset_root="/datasets/abarghi/"
+output_root="/tmp/samples"
```

Comment:
I'm guessing at a minimum we'll want these as params to the script?  a `FIXME:` is fine for now IMO.

Reply:
resolved